entries/t/tpg.com.au.json

```diff
@@ -0,0 +1,15 @@
+{
+  "TPG": {
+    "domain": "tpg.com.au",
+    "tfa": [
+      "sms"
+    ],
+    "documentation": "https://support.tpg.com.au/multi-factor-authentication-tpg",
```

Comment:
I'm getting an HTTP 403 Forbidden response to the entire `support.tpg.com.au` subdomain. Is it restricted to Australia or am I blocked for some reason by some CDN?